contracts/src/BeefyClient.sol

```diff
@@ -243,8 +265,14 @@ contract BeefyClient {
             revert NotEnoughClaims();
         }
 
-        tickets[createTicketID(msg.sender, commitmentHash)] =
-            Ticket(msg.sender, uint64(block.number), uint32(vset.length), 0, keccak256(abi.encodePacked(bitfield)));
+        tickets[createTicketID(msg.sender, commitmentHash)] = Ticket(
+            msg.sender,
+            uint64(block.number),
+            uint32(vset.length),
+            signatureSamples(vset.length, signatureCount),
+            0,
+            keccak256(abi.encodePacked(bitfield))
+        );
```

Comment:
I think there is an alternative syntax for this initializer which is more readable as it includes field names.

```solidity
Ticket ticket = Ticket({foo=1, foo=2})
```

Reply:
Addressed in https://github.com/Snowfork/snowbridge/pull/956/commits/4fb659255abf429d2157f413c4e00bf3dbb9b750